utils/utils.js

```diff
@@ -286,12 +286,17 @@ function getRevenue(properties, eventName) {
 function tranformToRudderNames(integrationObject) {
   Object.keys(integrationObject).forEach((key) => {
     if (integrationObject.hasOwnProperty(key)) {
-      if (commonNames[key]) {
-        integrationObject[commonNames[key]] = integrationObject[key];
+      const sanitizedIntgName = key.toLowerCase().trim();
```

Comment:
Not sure about this one, as I do not know the permutations of the key values.

It seems like only the display name is cast to lowercase in the utility method, for example NAME or any hardcoded CNameMapping coould have uppercase values.

If the key value in this context is alwys the display name this is fine elsewise toLowerCase would not work for all cases

Reply:
Yes, the keys in the common names map are always a combination of the display name. 

The casing is made to lower on both sides to eliminate any play w.r.t that. One would ideally override (rather, add new ones) the cname mapping for other combinations (ex: "Customer.io" could have other hardcoded names defined such as "customer-io", "kustomerio" (with a "K"), etc).

However, I've now fixed the bug (converted the case of even the hard-coded cname mappings)

Reply:
got it. LGTM